test/over_react/component/dom_components_test.dart

```diff
@@ -57,5 +62,5 @@ main() {
         expect(component.type, equalsIgnoringCase(expectedTagName));
       });
     }
-  });
+  }, tags: 'no-ddc');
```

Comment:
Hm, we should really still test these in the DDC if possible. 

What about if we enumerated these, and just checked them against the reflected values, like so?
```dart
const expectedStaticMethods = const <Symbol> [#div, #span, ...];
var actualMethods;
try {
  actualMethods = domClassMirror.staticMembers.values.map((m) => m.simpleName).toList();
} catch(e) {}
if (actualMethods != null) {
  expect(actualMethods, unorderedEquals(expectedStaticMethods),
      reason: '`expectedStaticMethods` needs to be updated');
}

for (var method in expectedStaticMethods) {
  String name = MirrorSystem.getName(method);
...
```